presto-spark-base/src/test/java/com/facebook/presto/spark/TestPrestoQueries.java

```diff
@@ -156,6 +156,12 @@ public void testExecuteUsingWithWithClause()
         // prepared statement is not supported by Presto on Spark
     }
 
+    @Override
+    public void testExplainValidate()
+    {
+        // prepared statement is not supported by Presto on Spark
```

Comment:
this is not a prepared statement.  Why doesn't explain(type validate) work anymore?

Reply:
I just wanted to skip this test from Spark, as we would need to build type validate support in the PrestoSparkQueryRunner. I was also wondering if this should even have type validate support.

Reply:
can we add this test.  We shouldn't exclude things from presto on spark support without good reason.

Reply:
1. This does not exclude this from presto on Spark, its only skipping from pos query runner which is only a test path unlike real pos path.
2. Since these queries are only analysis queries they are not differentiated between regular or pos.
3. I will make changes to pos query runner to support this test.